Makefile

```diff
@@ -0,0 +1,45 @@
+TEST=tests
+PDB_PATH=$(shell find /usr/lib{,64} -type f -executable -name pdb.py 2> /dev/null)
```

Comment:
maybe you can consider using "shell which pdb" here? 
In an virtualenv the solution as you suggested it is sadly failing.
"which pdb" is referencing to the first found "pdb" and thus gives for an virtualenv e.g. usr/bin/pdb which is also suggested by the documentation of ./test-module for -D

Reply:
As far as I can tell (centos7, system python, no virtualenv), the `-D` requires an executable `pdb.py`, not the executable found by running `which pdb`. Does it work for you with `which pdb` in a virtualenv?

Reply:
Also, you can try out different paths to see which one works with this syntax:

```
make debug MODULE=katello_content_view PDB_PATH=<whatever you want>
```

Reply:
yes it does - in principle. I cannot even start pdb by hand inside virtualenv.
maybe that is a debian problem?
although this is not really relevant at this place

Reply:
You know, perhaps it'd be pretty simple to just check if we're in a virtualenv and use `which pdb` in that case. Sound good?

Reply:
sounds good to me

Reply:
```
.tmp/ansible/hacking/test-module --help
<...>
  -D DEBUGGER, --debugger=DEBUGGER
                        path to python debugger (e.g. /usr/bin/pdb)
```
This sounds to me, that `<whatever_venv_or_not>/bin/pdb` should do the job. And that one shall always be `which pdb`.

Reply:
I agree that the documentation (what little there is) makes it sound like `which pdb` would work, but it does not work with native Python:

```sh
./.tmp/ansible/hacking/test-module -m modules/katello_content_view.py -a @tests/debug_data/content-view.json -D /usr/bin/pdb
* including generated source, if any, saving to: /home/vagrant/.ansible_module_generated
* ansiballz module detected; extracted module source to: /home/vagrant/debug_dir
Traceback (most recent call last):
  File "/usr/bin/pdb", line 7, in <module>
    from pdb.pdb import main
ImportError: No module named pdb
```

Reply:
Funny, in our setups it was the exact opposite. We got this error with the venv pdb, but the system wide one worked.